File: VersaoFinal/Latex/codigos/micro07.lua
local programa, numero, opc
programa = 1

while programa == 1 do
  io.write("Digite um numero: ")
  numero = tonumber(io.read())
  
  if numero > 0 then
    io.write("Positivo\n")
    
  else
    if numero == 0 then
      io.write("O numero é igual a 0\n")
    end
    if numero < 0 then
      io.write("Negativo\n")
    end
  end
  
  io.write("Deseja finalizar? (S/N): ")
  opc = io.read()
  
  if opc == 'S' then
    programa = 0
  end
end
      
      

File: CC/Codigos_Fonte_Relatorio_1/micro07/micro07.pas
program micro07;

var
    programa, numero : integer;
    opc : char;


begin
    programa := 1;

    while (programa = 1) do
    begin
        write('Digite um número: ');
        read(numero);   

        if(numero > 0) then
            writeln('Positivo')
        else
        begin
            if(numero = 0) then           
                writeln('O número é igual a zero')
            else 
                writeln('Negativo')
        end;
        
        write('Deseja finalizar? (S/N)');
        read(opc);

        if(opc = 'S') then        
            programa := 0
          
        
    end;



end.
